deeppavlov/metrics/accuracy.py

```diff
@@ -49,8 +49,13 @@ def sets_accuracy(y_true: [list, np.ndarray], y_predicted: [list, np.ndarray]) -
     Returns:
         portion of samples with absolutely coincidental sets of predicted values
     """
+    def _to_set(val):
+        if isinstance(val, (list, tuple, set)):
```

Comment:
I would argue that this change is unnecessary, as one can use simple `accuracy` metric for cases where labels are strings. Also there could be other iterable types that are not `list`, `tuple` or `set` that would not be converted to `set` this way but should be.

Reply:
Yes, I think, you are right, the proper way here is to use `accuracy` instead of `sets_accuracy` (it requires fixing `Proba2Labels` so that it returns just one label when `max_proba=True`).